packages/extractor-vue/README.md

```diff
@@ -0,0 +1,51 @@
+[![License][badge-license]][license]
+[![Version][badge-version]][package]
+[![Downloads][badge-downloads]][package]
+
+# @lingui/vue-extractor
+
+This package contains a custom extractor that handles Vue files. It supports extracting messages from script and setup scripts as well Vue templates.
+
+`@lingui/vue-extractor` is part of [LinguiJS][linguijs]. See the [documentation][documentation] for all information, tutorials and examples.
+
+## Installation
+
+```sh
+npm install @lingui/extractor-vue
+```
+
+## Usage
+
+This custom extractor requires that you use javascript for your lingui configuration.
```

Comment:
```suggestion
This custom extractor requires that you use JavaScript for your Lingui configuration.
```

Reply:
I'm thinking would it be better to write here something similar as we do in docs:

> To use custom extractor add to your `lingui.config.{js,ts}`